SrfGen/srfinfo2vm.py

```diff
@@ -455,20 +455,23 @@ def reduce_domain(origin, bearing, xlen, ylen, hh, space_srf, space_land, wd):
 
     xlen = math.ceil((dist_east_from_mid + dist_west_from_mid) / hh) * hh
 
-    if not np.isclose(over_n*(over_n > 0), over_s*(over_s > 0)):
+    over_n_max = over_n * (over_n > 0)
```

Comment:
Chuck a comment on this line to explain why we need to do these two lines and we are good to go.